docs/api-docs/channels/channels-storefront-tutorial.md

```diff
@@ -224,7 +166,7 @@ Accept: application/json
 
 ## Currencies settings
 
-Include the `Currencies` [protected section](#protected-ui-sections) in the channel's `config_meta` object to render the BigCommerce provided **Currencies** tab on the channel's settings page.
+Include the `currencies` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Currencies** tab on the channel's settings page.

```

Comment:
Same as above: get rid of tab and provide more clarity on this. Recommended: "Include the `currencies` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Currencies** menu item in the channel's navigation and corresponding settings page.